src/components/toast/toast-test.stories.tsx

```diff
@@ -254,3 +255,29 @@ export const ToastComponent = ({
     </>
   );
 };
+
+export const ToastWhenOtherModalRenders = () => {
```

Comment:
**suggestion(non-blocking):** Just thinking, this example might be a good candidate to have an implementation similar to [the issue we have open for Toast currently](https://github.com/Sage/carbon/issues/6410). I think all you'd need to do is add an additional `Dialog` that can be opened from another button.

Reply:
You can create this scenario by opening the Toast, closing and then re-opening the dialog. I think I'll just separate the Toast and Dialog here as I think adding another modal convolutes the example imo

Reply:
Tat's completely fine and I agree with this:

> I think I'll just separate the Toast and Dialog here as I think adding another modal convolutes the example imo